monai/transforms/spatial/dictionary.py

```diff
@@ -1311,6 +1338,29 @@ def __call__(self, data: Mapping[Hashable, np.ndarray]) -> Dict[Hashable, np.nda
             )
         return d
 
+    def inverse(self, data: Mapping[Hashable, np.ndarray]) -> Dict[Hashable, np.ndarray]:
+        d = deepcopy(dict(data))
+        for key, mode, padding_mode, align_corners in self.key_iterator(
+            d, self.mode, self.padding_mode, self.align_corners
+        ):
+            transform = self.get_most_recent_transform(d, key)
+            # Create inverse transform
+            zoom = np.array(transform[InverseKeys.EXTRA_INFO.value]["zoom"])
```

Comment:
I think `_do_transform` flag should be checked somewhere in this method?

Reply:
thanks, updated